hugo/layouts/partials/site/footer.html

```diff
@@ -1,4 +1,6 @@
 {{ $baseUrl := urls.Parse $.Site.Params.Baseurl }}
+{{ $issueUrl := print "https://github.com/cue-lang/docs-and-content/issues/new?labels=Triage,NeedsInvestigation,cuelang.org&template=issue_report.yml&cue-version=" .Permalink }}
```

Comment:
I think you can drop the labels from here, as they're in the template.

Reply:
Yeah, wasn't sure about that since the terraform example seemed to still add them as well. But I'll remove them!